Makefile

```diff
@@ -0,0 +1,119 @@
+	@echo "Installing external providers..."
+	mkdir -p ~/.llama/providers.d/inline/safety ~/.llama/providers.d/remote/tool_runtime
+	wget -q https://raw.githubusercontent.com/lightspeed-core/lightspeed-providers/refs/heads/main/resources/external_providers/inline/safety/lightspeed_question_validity.yaml \
+	  -O ~/.llama/providers.d/inline/safety/lightspeed_question_validity.yaml
+	wget -q https://raw.githubusercontent.com/lightspeed-core/lightspeed-providers/refs/heads/main/resources/external_providers/remote/tool_runtime/lightspeed.yaml \
+	  -O ~/.llama/providers.d/remote/tool_runtime/lightspeed.yaml
+	. venv/bin/activate && pip install lightspeed_stack_providers
+	@echo "External providers installed."
+
+check-faiss-db:
+	@if [ ! -f aap_faiss_store.db ]; then \
+		echo "$(RED)Warning: aap_faiss_store.db not found in the repository root."; \
+		echo "Please copy the aap_faiss_store.db file to the repository root before building.$(NC)"; \
+	else \
+		echo "aap_faiss_store.db found."; \
+	fi
+
+build: check-faiss-db
+	@echo "Building base Ansible Chatbot Stack image..."
+	export LLAMA_STACK_LOGGING=server=debug;core=info && \
+	export UV_HTTP_TIMEOUT=120 && \
+	. venv/bin/activate && \
+	llama stack build --config ansible-chatbot-build.yaml --image-type container
+	@echo "Base image $(RED)ansible-chatbot-stack-base$(NC) built successfully."
+
+build-custom: build
+	@echo "Building customized Ansible Chatbot Stack image..."
+	docker build -f Containerfile -t ansible-chatbot-stack:$(ANSIBLE_CHATBOT_STACK_VERSION) --build-arg ANSIBLE_CHATBOT_STACK_VERSION=$(ANSIBLE_CHATBOT_STACK_VERSION) .
+	@echo "Custom image $(RED)ansible-chatbot-stack:$(ANSIBLE_CHATBOT_STACK_VERSION)$(NC) built successfully."
+
+run:
+	@echo "Running Ansible Chatbot Stack container..."
+	@echo "Using vLLM URL: $(ANSIBLE_CHATBOT_VLLM_URL)"
+	@echo "Using inference model: $(ANSIBLE_CHATBOT_INFERENCE_MODEL)"
+	docker run --security-opt label=disable -it -p $(LLAMA_STACK_PORT):$(LLAMA_STACK_PORT) \
+	  --env LLAMA_STACK_PORT=$(LLAMA_STACK_PORT) \
+	  --env VLLM_URL=$(ANSIBLE_CHATBOT_VLLM_URL) \
+	  --env VLLM_API_TOKEN=$(ANSIBLE_CHATBOT_VLLM_API_TOKEN) \
+	  --env INFERENCE_MODEL=$(ANSIBLE_CHATBOT_INFERENCE_MODEL) \
+	  ansible-chatbot-stack:$(ANSIBLE_CHATBOT_STACK_VERSION)
+
+clean:
+	@echo "Cleaning up..."
+	rm -rf providers.d/
+	@echo "Removing ansible-chatbot-stack images..."
+	docker rmi -f $$(docker images -a -q --filter reference=ansible-chatbot-stack) || true
+	@echo "Removing ansible-chatbot-stack-base image..."
+	docker rmi -f $$(docker images -a -q --filter reference=ansible-chatbot-stack-base) || true
+	@echo "Clean-up complete."
+
+deploy-k8s:
+	@echo "Deploying to Kubernetes cluster..."
+	kubectl apply -f ansible-chatbot-deploy.yaml
+	@echo "Deployment initiated. Verify using kubectl commands."
+
+shell:
+	@echo "Getting a shell in the container..."
+	docker run --security-opt label=disable -it --entrypoint /bin/bash ansible-chatbot:$(ANSIBLE_CHATBOT_VERSION)
```

Comment:
I think the image name is [now](https://github.com/ansible/ansible-chatbot-stack/pull/9/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R74) `ansible-chatbot-stack`?